diff --git a/src/terminal/stream.zig b/src/terminal/stream.zig
index 9771334..6e7366b 100644
--- a/src/terminal/stream.zig
+++ b/src/terminal/stream.zig
@@ -19,7 +19,7 @@ const sgr = @import("sgr.zig");
 const UTF8Decoder = @import("UTF8Decoder.zig");
 const MouseShape = @import("mouse.zig").Shape;
 
-const log = std.log.scoped(.stream);
+const log = std.log.scoped(.libghostty_vt_stream);
 
 /// Flip this to true when you want verbose debug output for
 /// debugging terminal stream issues. In addition to louder
@@ -1380,7 +1380,7 @@ pub fn Stream(comptime H: type) type {
                         if (modes.modeFromInt(mode_int, ansi_mode)) |mode| {
                             self.handler.vt(.set_mode, .{ .mode = mode });
                         } else {
-                            log.warn("unimplemented mode: {}", .{mode_int});
+                            log.debug("unimplemented mode: {}", .{mode_int});
                         }
                     }
                 },
@@ -1401,7 +1401,7 @@ pub fn Stream(comptime H: type) type {
                         if (modes.modeFromInt(mode_int, ansi_mode)) |mode| {
                             self.handler.vt(.reset_mode, .{ .mode = mode });
                         } else {
-                            log.warn("unimplemented mode: {}", .{mode_int});
+                            log.debug("unimplemented mode: {}", .{mode_int});
                         }
                     }
                 },
